docker-compose.yml

```diff
@@ -540,6 +540,38 @@ services:
       - triplestore:virtuoso
     logging: *default-logging
 
+  vendor-management-consumer:
```

Comment:
Try using this setup. No remap is needed, so lees complicated consumer.
Ensure that you use the latest versions
https://github.com/lblod/app-worship-decisions-database/blob/master/docker-compose.yml#L266-L284